R/extendr-wrappers.R

```diff
@@ -1252,7 +1252,9 @@ RPolarsSeries$to_frame <- function() .Call(wrap__RPolarsSeries__to_frame, self)
 
 RPolarsSeries$set_sorted_mut <- function(descending) invisible(.Call(wrap__RPolarsSeries__set_sorted_mut, self, descending))
 
-RPolarsSeries$from_arrow <- function(name, array) .Call(wrap__RPolarsSeries__from_arrow, name, array)
+RPolarsSeries$from_arrow_array_stream_str <- function(name, robj_str) .Call(wrap__RPolarsSeries__from_arrow_array_stream_str, name, robj_str)
```

Comment:
I was going to use this as the inside of `as_polars_series`, but now I can't because it causes a segmentation fault.
See https://github.com/pola-rs/r-polars/issues/732#issuecomment-1916599082.